DNN Platform/Modules/DDRMenu/MenuBase.cs

```diff
@@ -374,7 +410,7 @@ private void ApplyNodeManipulator()
             this.RootNode =
                 new MenuNode(
                     ((INodeManipulator)Activator.CreateInstance(BuildManager.GetType(this.menuSettings.NodeManipulator, true, true))).
-                        ManipulateNodes(this.RootNode.Children, this.HostPortalSettings));
+                        ManipulateNodes(this.RootNode.Children, (PortalSettings)this.HostPortalSettings));

```

Comment:
Isn't `HostPortalSettings` still returning `PortalSettings`?
```suggestion
                        ManipulateNodes(this.RootNode.Children, this.HostPortalSettings));
```

Reply:
Addressed in https://github.com/dnnsoftware/Dnn.Platform/pull/4224/commits/45da40c51e44f89fefa2c67929141f5861054139